### ISSUE TITLE
The remote server returned an error: (522) Origin Connection Time-out.

### ISSUE THREAD

**Issue by PaulMPomichter** [EasyFarm/EasyFarm#485]
Getting this error when I run EasyFarm, worked fine earlier, and didn't change anything since

**Comment by rikiyame** [comment]
Hi Paul, i am having same issues just crashed was running for hours earlier no problem.

**Comment by ZirkPrime** [comment]
Yes exactly the same for me, was working for friend and myself earlier, now getting the 522 error.
